package.json

```diff
@@ -69,7 +69,6 @@
         "webpack-merge": "^5.10.0"
     },
     "dependencies": {
-        "@types/node": "^22.7.5",
```

Comment:
Does it work if this is 18 instead of 22? 

I'm confused about the relationship of the copyFiles.ts scripts vs the packages/x/package.json declarations.

As an alternative to `fs as any`, maybe we could use the `techdebt.test.ts` reminder to remind us to remove this top-level dependency when the `getMinNodejsVersion` result is new enough?

Reply:
Looks like that works as well, I can update the techdebt test to match. I thought avoiding the dependency was more important than the `any` workaround. But I guess the version bump isn't something to worry about?

Reply:
> I thought avoiding the dependency was more important than the `any` workaround.

Since this PR makes the total cost clearer (i.e.: small :), I'm having doubts. And also, since the risk is covered by techdebt.test.ts in either case, perhaps it's better to have type checking.

Either way is fine IMO.

Reply:
I think I'll keep the dependency then. Since its just a version bum (to an old stable version) and not a whole new thing, the cost seems very low. The `any` code doesn't read well, and could cause confusion to someone without the necessary context.